CMakeLists.txt

```diff
@@ -285,6 +285,7 @@ set(SIMPLE_MD_SOURCES
 	"simplemd/configurations/RDFConfiguration.cpp"
 	"simplemd/configurations/SimulationConfiguration.cpp"
 	"simplemd/configurations/VTKConfiguration.cpp"
+  "simplemd/configurations/DomainDecompConfiguration.cpp"
```

Comment:
Formatting (indentation)

Reply:
This part in in tabs for some reason, while the rest is in spaces